### PR TITLE
Separate deployment of webapp and worker

### PR DESCRIPTION
**Pull request by saliceti**
# Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-856

## Changes in this PR:
To be able to monitor the worker, webapp and worker should be 2 different apps.
This PR contains the new manifests and scripts for the migration.

## How to test
I can demo this on staging

### CODE REVIEW

**Comment by cesidio** [comment]
We need to update the documentation about GOV.UK PaaS now that we do not need the beta version of CloudFoundry and we need to set up environment variables for web and worker instance

File: bin/separate-app-worker-2 [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# cf start "teaching-vacancies-$ENVIRONMENT-TEMP"
+
+echo Send traffic to temp webapp
+cf map-route "teaching-vacancies-$ENVIRONMENT-TEMP" london.cloudapps.digital --hostname "teaching-vacancies-$ENVIRONMENT"
+
+echo Delete old webapp
+cf delete "teaching-vacancies-$ENVIRONMENT" -f
+
+echo Deploy new webapp
+cf push -f "paas/web/manifest-$ENVIRONMENT.yml" --no-start teaching-vacancies-$ENVIRONMENT
```
Comment (cesidio) [inline bin/separate-app-worker-2:20]: you need to rename `paas/web/manifest.yml` and `paas/worker/manifest.yml` to `paas/web/manifest-production.yml` and `paas/worker/manifest-production.yml`

File: bin/separate-app-worker-2 [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ENVIRONMENT=$1
+
+# echo Start new worker
+# cf start "teaching-vacancies-worker-$ENVIRONMENT"
+
+# echo Start temp webapp
+# cf start "teaching-vacancies-$ENVIRONMENT-TEMP"
+
+echo Send traffic to temp webapp
+cf map-route "teaching-vacancies-$ENVIRONMENT-TEMP" london.cloudapps.digital --hostname "teaching-vacancies-$ENVIRONMENT"
```
Comment (cesidio) [inline bin/separate-app-worker-2:14]: Sometimes you use `cf` sometimes, `cf7`. We should make it consistent

**Comment by saliceti** [comment]
> We need to update the documentation about GOV.UK PaaS now that we do not need the beta version of CloudFoundry and we need to set up environment variables for web and worker instance

done

**Pull request merged**